client/src/pages/Game.js

```diff
@@ -28,20 +25,27 @@ function Game() {
   const handleQuestionChange = function (questionID) {
     setCurrQuestion(questionID);
 
-    setShow(!show);
+    // Update collection of questions that have been answered
+    setAnsweredQuestions(prevAnsweredQuestions => ({
```

Comment:
This is the core logic that powers question buttons disabling after selection. After a button is clicked, I add the ID to an object. Later, when deciding if a button should be disabled, I look at this map and see if the ID of the question related to the button is present in the map; if present, the question has been accessed before and thus should be disabled.